### DISCUSSION TITLE
Panel Resize: legend table is not responsive when panel is resized

### DISCUSSION THREAD

**Discussion by MacNale** [grafana/grafana#38380] [Dashboards, Panels & Visualizations]
**What happened**:
I have a panel with legend->values like min, max, avg, total, current shown as right side table in a panel. When I resize the panel, it does not adjust the side legend table nicely.

![image](https://user-images.githubusercontent.com/14235758/82632843-0600dc80-9bbf-11ea-8142-a068a74eeffc.png)

**What you expected to happen**:
This is not a high severity issue. BUT misses the perfection Grafana had. Panel needs to be responsive.

**How to reproduce it (as minimally and precisely as possible)**:
Add legend values as right hand table (Min, Max, Current, Total, avg) and resize the panel by narrowing the length.

![image](https://user-images.githubusercontent.com/14235758/82633011-993a1200-9bbf-11ea-818c-bbb7cc1d8f9e.png)

**Anything else we need to know?**:

**Environment**:
- Grafana version: 7.0.0
- Data source type & version: NA
- OS Grafana is installed on: Docker
- User OS & Browser: Chrome
- Grafana plugins: Graph/anything

**Comment by torkelo** [comment]
Legend in table mode is not responsive, it takes up whatever space it needs given the number of values (columns) you have selected.

There is no easy way to make table responsive in this case other than changing the layout to move the legend under the graph. Is this what you wish Grafana did automatically? Ignore the "Toi the right" option and move it under when the width is too low?

**Comment by MacNale** [comment]
Moving under the graph is not a bad idea.

**Comment by daniellee** [comment]
When making the window smaller rather than resizing, the legend does go under the graph.

Large window:
![image](https://user-images.githubusercontent.com/434655/98940853-eb634c80-24eb-11eb-9c55-879e9b5e3f68.png)

With a smaller window, the responsive design moves the legend under the graph:
![image](https://user-images.githubusercontent.com/434655/98940870-f4541e00-24eb-11eb-9b34-015b97d2066a.png)

Maybe we can apply the same styling to resizing but would need some UX advice on whether that is a good idea or not.

**Reply by xv-francesco** [reply]
This doesn't seem to happen in the new Time Series panel, where the legend stays on the right even when resizing the window. Is this worth a new discussion thread?

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
